README.md

```diff
@@ -1,6 +1,149 @@
 # Keycloak-deployment
 
-
 This repository contains all our scripts to deploy keycloak on Openshift and minishift.
 Also we have scripts to bake our own docker image using the keycloak source code
 from our repository `almighty/keycloak`.
+
+# Almighty-Keycloak Docker Image
+
+To build this image it is necessary to have previously generated the executables of this
+project. To do this, run the following Maven command in the almighty/keycloak repository:
+
+`$ mvn clean install -DskipTests -pl :keycloak-server-dist -am -P distribution`
```

Comment:
Here we are building the keycloak-server-dist, but if it is the first time you run this project. You need `$ mvn clean install -DskipTests -am -P distribution`